docker/developers/docker-compose.minio.yml

```diff
@@ -0,0 +1,113 @@
+include:
+  - path: ./docker-compose.networks.yml
+
+services:
+  minio-base: &minio-base
+    hostname: minio
+    # x86 image:
+    image: minio/minio:${MINIO_VERSION:-latest}
+    # arm image:
+    #image: pixelchrome/minio-arm:latest-arm32v7
+    volumes:
+      - minio1_storage:/data1
+      - minio2_storage:/data2
+      - minio3_storage:/data3
+      - minio4_storage:/data4
+    command: server --console-address ":3001" /data1 /data2 /data3 /data4
+
+  minio:
+    <<: *minio-base
+    environment: &minio-env
+      MINIO_VOLUMES: "http://localhost:9000/data{1...4}"
```

Comment:
Will using port 9000 here conflict with our use of port 9000 for webpack dev server here?

https://github.com/code-dot-org/code-dot-org/blob/093b289ffc52a81d1e69c83610d776da9436fbeb/apps/webpack.config.js#L33